### DISCUSSION TITLE
Can OpenSSL convert SSH keys into PKCS #8 (PEM)?

### DISCUSSION THREAD

**Discussion by christian-2** [openssl/openssl#23442] [Q&A] [answered]
I am trying to convert an SSH key of type `ed25519` (`ssh-ed25519 ...`) into PKCS #8 (PEM) with OpenSSL 3.0.11 and by invoking `ssh-keygen -f $file -e -m PKCS8`. However, I receive that error: "do_convert_to_pkcs8: unsupported key type ED25519". Is this a missing feature or perhaps a restriction that is gone in the latest version of OpenSSL? (OpenSSL 3.0.11 is the default version in the current version 12.4 of Debian.)

**Comment by levitte** [comment, marked as answer]
For this, you should talk with the OpenSSH folks.  We are not them.

https://www.openssh.com/

**Comment by christian-2** [comment]
@levitte Ah, yes of course. My bad, sorry.
